src/pages/User/TermCondition.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <q-page>
+    <div class="q-pa-md q-gutter-sm">
+      <q-breadcrumbs>
+        <q-breadcrumbs-el label="Home" to="/" />
+        <q-breadcrumbs-el label="Term and Conditon" />
+      </q-breadcrumbs>
+    </div>
+    <div class="row q-pa-xl">
+      <div class="col-8">
+        <div class="q-mb-xl">
+          <p class="text-h3"> Term and Condition</p>
+          <div>
+            <p>PLEASE NOTE THAT YOUR USE OF AND ACCESS TO OUR SERVICES (DEFINED BELOW) 
+            ARE SUBJECT TO THE FOLLOWING TERMS. IF YOU DO NOT AGREE TO ALL OF THE FOLLOWING, 
+            YOU MAY NOT USE OR ACCESS THE SERVICES IN ANY MANNER.</p>
+            <p class="text-h5"> Term of Use </p>
+            <p> Effective on 39 February 2021 </p>
+            <p> Welcome to Kaggle. Please read on to learn the rules and restrictions that govern your use of our website(s), 
+            products, services and applications (the “Services”).
+            If you have any questions, comments, or concerns regarding these Terms or the Services, please contact us. 
+            These Terms of Use (the “Terms”) are a binding contract between you and Kaggle Inc. (“Kaggle,” “we” and “us”). 
+            You must agree to and accept all of the Terms, or you don’t have the right to use the Services. 
+            Your use of the Services in any way means that you agree to all of these Terms, and these Terms will remain in effect while you use the Services. 
+            These Terms include the provisions in this document, as well as those in the Privacy Policy and Copyright Dispute Policy. 
+            In these Terms, the words “include” or “including” mean “including but not limited to”, 
+            and examples are for illustration purposes and are not limiting. </p>
+            <p class="text-h6"> Will these Terms ever change? </p>
+            <p> We are constantly trying to improve our Services, so these Terms may need to change along with the Services. We reserve the right to change the Terms at any time, 
+            but if we do, we will bring it to your attention by placing a notice on the www.kaggle.com website, by sending you an email, or by some other means. </p>
+          </div>
```

Comment:
Please remove this part and leave it empty. Its a bad practice to put in other "company" legal in open source.